spec/controllers/v0/institutions_controller_spec.rb

```diff
@@ -59,9 +67,11 @@
   context 'search results' do
     before(:each) do
       create(:version, :production)
-      2.times { create(:institution, :in_nyc) }
-      create(:institution, :in_chicago, online_only: true)
-      create(:institution, :in_new_rochelle, distance_learning: true)
+      2.times { create(:institution, :in_nyc, approved: true) }
+      create(:institution, :in_chicago, online_only: true, approved: true)
+      create(:institution, :in_new_rochelle, distance_learning: true, approved: true)
+      # adding a non approved institutions row
+      create(:institution, :contains_harv, approved: false)
```

Comment:
This is good here because in the next spec (on line 79) it is shown that adding this did not affect the number of returned rows.